README.md

```diff
@@ -140,16 +204,44 @@ storage:
+- teamcity/code-quality: simply run `tox` in the top-level dir to run all syntax checks as well as pytest (unit-tests). See [tox.ini](https://github.com/dcos/dcos/blob/master/tox.ini) for more details
+- integration-test/*: runs [integration_test.py](https://github.com/dcos/dcos/blob/master/test_util/integration_test.py) in the network of a DC/OS cluster
     - /vagrant-bash: Tests the on-prem bash provider by using [dcos-vagrant](https://github.com/dcos/dcos-vagrant). Invoke this test through [run-all](https://github.com/dcos/dcos/blob/master/test_util/run-all)
     - /deploy-vpc-cli: runs [ccm-deploy-test](https://github.com/dcos/dcos/blob/master/test_util/test_installer_ccm.py) with USE_INSTALLER_API=false. A Virtual Private Cloud of centos nodes is spun up by CCM (Mesosphere's Cloud Cluster Manager) and the installer (dcos_generate_config.sh) is used via the CLI options to deploy DC/OS. Finally, the same integration_test.py is run
     - /deploy-vpc-api: the same as /deploy-vpc-cli (see above) except uses USE_INSTALLER_API=true, which causes the installer to be started with the `--web` option and then controlled entirely by the HTTP API
 
+
+# Repo Structure
+
+DC/OS itself is composed of many individual components precisely configured to work together in concert.
+
+This repo contains the release and package building tools necessary to produce installers for various on-premises and cloud platforms.
+
+| Directory | Contents |
+| --------- | -------- |
+| *docker*             | Locally defined docker containers packages are built in
+| *docs*               | Documentation
+| *dcos_installer*     | Backend for Web, SSH, and some bits of the Advanced installer. Code is being cleaned up
+| *gen*                | Python library for rendering yaml config files for various platforms into packages, with utilities to do things like make "late binding" config set by CloudFormation
+| *gen/build_deploy*   | Code to take a build and transform it into a particular platform deployment tool (Bash / command line, AWS, Azure, etc.)
+| *packages*           | Packages which make up DC/OS (Mesos, Marathon, AdminRouter, etc). These packages are built by pkgpanda, and combined into a "bootstrap" tarball for deployment.
+| *pkgpanda*           | DC/OS baseline/host package management system. Tools for building, deploying, upgrading, and bundling packages together which live on the root filesystem of a machine / underneath Mesos.
+| *pytest*             | Misc. tests. Should be moved to live next to the appropriate code
+| *release*            | Release tools for DC/OS. (Building releases, building installers for releases, promoting between channels)
+| *ssh*                | AsyncIO based parallel ssh library used by the installer
+| *test_util*          | various scripts, utilities to help with integration testing
+
+All code in this repository is Python 3
```

Comment:
Contextual. The intention, I assume, was to differentiate Python 2 usage vs Python 3.
I'd be okay with killing this line.